### PR TITLE
Update README.md with Index and Status

### PR DESCRIPTION
**Pull request by SpockBauru**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,15 @@ English translation project for Koikatsu Sunshine
 
 The translations are applied while the game is running and do not require replacing or modifying any game files.
 
+## Index
+ * [Prerequisites](#prerequisites)
```
Comment (GeBo1) [inline README.md:8]: Prefer using Contents here (also you should have a blank line after a heading)

```suggestion
## Contents

 * [Prerequisites](#prerequisites)
```

File: README.md [hunk 2]
```diff
@@ -274,18 +283,36 @@ Some prefixes can be combined with numbers, which limit them to matching specifi
...
 
+
 ## Status
 
-Right now the project is at a fast pace and things are constantly changing. Coordinate with other translators on the [Koikatsu Discord](https://discord.gg/urDt8CK) #lingual-studies channel. To avoid translation conflicts please ask if anyone is working on a file. If you have any questions about the quality of your translations, ask for advice on the server.
+Right now 37% of the game is translated and we are prioritizing the UI and game items. The project is evolving at a fast pace and things are constantly changing, so coordinate with other translators in the [Koikatsu Discord](https://discord.gg/urDt8CK) #lingual-studies channel. To avoid translation conflicts please take a look in the To Do list on #whiteboard channel and ask if anyone is working on a file in #lingual-studies channel. If you have any questions about the quality of your translations, ask for advice on the server.
+
+### To Do List
+
+**Translate Tutorials:** Delete `KoikatsuSunshine\UserData\save\Global.dat` to make the tutorials appear again (make a backup before!)
```
Comment (GeBo1) [inline README.md:302]:
```suggestion
#### Translate Tutorials

To view the tutorials either press `F5` while playing the main game or delete `KoikatsuSunshine\UserData\save\Global.dat` to make the tutorials appear again (make a backup before!)
```

File: README.md [hunk 3]
```diff
@@ -274,18 +283,36 @@ Some prefixes can be combined with numbers, which limit them to matching specifi
...
+
+- <s>Translate Chara Maker Tutorial</s> Taken by xmex
```
Comment (GeBo1) [inline README.md:304]:
```suggestion
- ~~Translate` Chara Maker Tutorial~~ Taken by xmex
```

File: README.md [hunk 4]
```diff
@@ -274,18 +283,36 @@ Some prefixes can be combined with numbers, which limit them to matching specifi
...
+- Translate Free-H Tutorial
+- Translate H-Scenes Tutorial
+- Translate Talking to the Girl Tutorial
+- Translate Talking with Hikari Tutorial
+- Translate Oil Minigame Tutorial
+
+**Translate the `RedirectedResources` folder:**
```
Comment (GeBo1) [inline README.md:311]:
```suggestion
#### Translate the `RedirectedResources` folder
```

File: README.md [hunk 5]
```diff
@@ -274,18 +283,36 @@ Some prefixes can be combined with numbers, which limit them to matching specifi
...
+See the readme above on how to handle the MachineTranslations. The goal is to manually translate everything in the files `translation.txt` and clear all `zz_MachineTranslation.txt` files.
+
+- <s>Check Nicknames in `RedirectedResources\assets\abdata\etcetra\list\nickname\00`</s> Taken by Dark Pulse
```
Comment (GeBo1) [inline README.md:314]:
```suggestion
- ~~Check Nicknames in `RedirectedResources\assets\abdata\etcetra\list\nickname\00`~~ Taken by Dark Pulse 
```

**Review by ManlyMarco — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (SpockBauru) [reply README.md:304]: Double tilde is not working in the preview, even if it appear in the "edit" window. I also tried a handful of Strikethrough commands and the only one that worked was the <s></s>

File: README.md (hunk 5 above)
Comment (SpockBauru) [reply README.md:314]: Double tilde is not working

File: README.md (hunk 3 above)
Comment (GeBo1) [reply README.md:307]: That's odd, it's in the github documentation, but ok.

**Review by GeBo1 — APPROVED** [review]

**Pull request merged**